packages/web-components/src/divider/divider.styles.ts

```diff
@@ -0,0 +1,125 @@
+import { css } from '@microsoft/fast-element';
+import {
+  colorBrandForeground1,
+  colorBrandStroke1,
+  colorNeutralForeground1,
+  colorNeutralForeground2,
+  colorNeutralForeground3,
+  colorNeutralStroke1,
+  colorNeutralStroke2,
+  colorNeutralStroke3,
+  fontFamilyBase,
+  fontSizeBase200,
+  fontWeightRegular,
+  strokeWidthThin,
+} from '../theme/design-tokens.js';
+
+/** Divider styles
+ * @public
+ */
+export const styles = css`
+  :host {
+    display: flex;
+  }
+
+  :host::after,
+  :host::before {
+    align-self: center;
+    background: ${colorNeutralStroke2};
+    box-sizing: border-box;
+    content: '';
+    display: flex;
+    flex-grow: 1;
+    height: ${strokeWidthThin};
+  }
+
+  :host([inset]) {
+    padding: 0 12px;
+  }
+
+  :host ::slotted(*) {
+    color: ${colorNeutralForeground2};
+    font-family: ${fontFamilyBase};
```

Comment:
@chrisdholt  Do we have a single alias for the font that sets family, weight, size, etc? Spec calls for `caption`. Do have an alias like fontCaption1?

Reply:
For now I would just apply the tokens. In many cases I’ve seen it’s not applying all those styles. We can create partials but that’s something I think can be a follow up if we need it.